src/overlays/actors/ovl_En_Tru/z_en_tru.c

```diff
@@ -1,3 +1,9 @@
+/*
+ * File: z_en_tru.c
+ * Overlay: ovl_En_Tru
+ * Description: Koume on Broom
```

Comment:
```suggestion
 * Description: Koume in Woods of Mystery
```
Judging by the fact that `NA_SE_EN_KOUME_DRINK` and `NA_SE_EN_KOUME_REGAIN` sfx's are in here, I think this is safe to assume this involves code when Koume is not on the broom and is in the woods of mystery injured on the ground. The other Koume actor, `ovl_En_Tru_Mt` is for Koume's Target Game. I'm not sure which of these actors has Koume in the boat-shop, but that can worked out afterwards

Reply:
Your assumption is correct:
![NKZE01_2021-09-27_23-19-40](https://user-images.githubusercontent.com/12245827/135033803-5494fe86-f83e-4143-9602-46d479267184.png)

Though the same actor rides the broom once you revive her:
![NKZE01_2021-09-27_23-22-17](https://user-images.githubusercontent.com/12245827/135034000-05063e8b-78da-482d-8df2-ad32b3dd08af.png)

Funnily enough, the Koume in the boat shop is En_Dnh, which is currently called "Boat Cruise Target Spot"